### PR TITLE
Fix println built-in function output to standard error

### LINKED ISSUE #2723: BUG: sealos create out is stderr
### Sealos Version

main

### How to reproduce the bug?

```
 sealos create --short b5f854a1cbeb  >stdout  2>stderr
root@default-golang-0:~/go/src/github.com/labring/runtime-cache# cat stderr
/var/lib/containers/storage/overlay/0b9b80d506f99ef98f27b4a3a52378841338e7d4bfa37f73206bb583256af766/merged
root@default-golang-0:~/go/src/github.com/labring/runtime-cache# cat stdout
```

### What is the expected behavior?

_No response_

### What do you see instead?

_No response_

### Operating environment

```markdown
- Sealos version:
- Docker version:
- Kubernetes version:
- Operating system:
- Runtime environment:
- Cluster size:
- Additional information:
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bxy4543**
1. The print built-in function formats its arguments in an implementation-specific way and writes the result to standard error.
Use `fmt.Println` instead.  fix #2723 
2. change docs: `kubectl get secrets ${UserName}.${CRDName}.credentials.postgresql.acid.zalan.do --template '{{.data.password}}' | base64 -d`

### CODE REVIEW

File: pkg/buildah/create.go [hunk 1]
```diff
@@ -72,7 +73,8 @@ func newCreateCmd() *cobra.Command {
 			if !opts.short {
 				logger.Info("Mount point: %s", info.MountPoint)
 			} else {
-				println(info.MountPoint)
+				println()
```
Comment (cuisongliu) [inline pkg/buildah/create.go:76]: Is there too much written here?

File: staging/src/github.com/labring/image-cri-shim/pkg/cri/cri_test.go [hunk 2]
```diff
@@ -26,5 +26,5 @@ func TestNewContainerRuntime(t *testing.T) {
 	//driver,_:=cr.processConfigFile()
 	//t.Log(driver)
 	cc := []string{"aa", "bb", "cc"}
-	println(fmt.Sprintf("%+v", cc))
+	fmt.Println(fmt.Sprintf("%+v", cc))
```
Comment (zzjin) [inline staging/src/github.com/labring/image-cri-shim/pkg/cri/cri_test.go:29]: Replace dupicate usage of  printf.
```suggestion
	fmt.Println("%+v", cc)
```

Comment (bxy4543) [reply staging/src/github.com/labring/image-cri-shim/pkg/cri/cri_test.go]: done

File: pkg/buildah/create.go (hunk 1 above)
Comment (bxy4543) [reply pkg/buildah/create.go]: done

**Review by zzjin — APPROVED** [review]

**Pull request merged**